Source/OCMock/OCMStubRecorder.h

```diff
@@ -60,6 +60,18 @@
 
 @property (nonatomic, readonly) OCMStubRecorder *(^ _ignoringNonObjectArgs)(void);
 
+// Break to the debugger when stub/expectation is called
+#define andDebugger() _andDo(^(NSInvocation *_invocation)             \
+{                                                                     \
+  __builtin_debugtrap();                                              \
```

Comment:
I get the intent, but I think this implementation is too simplistic. The problem, for me, is that this will break the debugger at the line where the `andDebugger()` macro is used, and not where the action is triggered.

Reply:
In the debugger you get the backtrace though that shows you where the action was triggered.

See the attached screen shots. 

![Before](https://user-images.githubusercontent.com/320201/89925808-cabbeb80-dbb8-11ea-8d13-83291acc6970.png)
![After](https://user-images.githubusercontent.com/320201/89925809-cb548200-dbb8-11ea-9e46-dc0c6b8b6e32.png)

I actually find this more useful because if it just broke where the action was triggered I wouldn't be sure which debugger statement actually triggered the stop. Perhaps I'm misunderstanding what you are meaning?